src/Query/QueryResult.php

```diff
@@ -485,13 +485,35 @@ public function toArray() {
 			'meta'=> [
 				'hash'   => md5( json_encode( $serializeArray ) ),
 				'count'  => $this->getCount(),
+				'total'  => $this->getTotalCount(),
 				'offset' => $this->mQuery->getOffset(),
 				'source' => $this->mQuery->getQuerySource(),
 				'time'   => number_format( ( microtime( true ) - $time ), 6, '.', '' )
 				]
 			]
 		);
 	}
+	
+	/**
+	 * Returns the total number of query results regardless of maximum to retrieval
+	 * 
+	 * @since ...
+	 *
+	 * @return integer|null
+	 */
+	public function getTotalCount() {
```

Comment:
Please use actual return type syntax rather than docs

getTotalCount(): int? {

Reply:
Thanks for the quick response! Type declaration does seem to be preferable.